e2e/app/fund.go

```diff
@@ -53,9 +52,9 @@ func accountsToFund(network netconf.ID) []common.Address {
 // fundAccounts funds the EOAs that need funding (just on anvil chains, for now).
 func fundAccounts(ctx context.Context, def Definition) error {
 	accounts := accountsToFund(def.Testnet.Network)
-	eth100 := math.NewInt(100).MulRaw(params.Ether) // 100 ETH
+	eth100 := new(big.Int).Mul(big.NewInt(params.Ether), big.NewInt(100))
```

Comment:
I see we also use `.NewInt()` then `.BigInt()` in `valupdates.go` and `evm.go`. 

Would changing it there be better too?